charts/tensorleap/charts/engine/templates/engine-job-template-cm.yaml

```diff
@@ -12,6 +12,9 @@ data:
       ttlSecondsAfterFinished: 36000 # 10 hours
       backoffLimit: 0
       template:
+        metadata:
+          annotations:
+            ad.datadoghq.com/engine.logs: '[{"source": "python", "service": "engine"}]'
```

Comment:
Currently we're getting a lot of "spam" logs from engine. We're getting a large amount of warnings and info logs from packages that are arriving as errors, could we filter them here?

Reply:
Log filtering can be done crudely at the agent level and it can be done more accurately during ingestion.
I suggest we do it during ingestion because:
1. It's easier and can be configured centrally without deploying on-prem changes.
2. It's not really a matter of cost, but rather a matter of noise, and doing it during ingestion is easier to tweak and fine-tune.
3. It can be done after json parsing so it's more robust and reliable. Less change of losing important data.

Reply:
@YuvalFeldman these are what seem to be the main culprits when it comes to `engine` logs: 
<img width="1446" alt="image" src="https://github.com/tensorleap/helm-charts/assets/233414/9678d854-e87a-4982-a40c-83951b01700f">

Would you like me to try to filter those out during ingestion?